package.json

```diff
@@ -20,7 +21,7 @@
     "lint": "yarn lint:es && yarn lint:style",
     "lint:es": "eslint 'packages/*/src/**/*.js'",
     "lint:style": "stylelint 'packages/*/src/**/*.scss' --report-needless-disables --report-invalid-scope-disables",
-    "test": "jest",
+    "test": "yarn run audit && jest",
```

Comment:
I'm wondering what your thoughts on keeping the audit and test scripts separate in `package.json`, and instead having a step specifically for auditing in CI? https://github.com/carbon-design-system/ibm-cloud-paks/blob/master/.github/workflows/ci-check.yml#L12

Reply:
hmmm, yeah, I don't mind that either. I'll factor it in, cheers for the proposal 👍